test/test_utils/test_validate.py

```diff
@@ -1,15 +1,22 @@
+import sys
 import unittest
 import logging
 
 import numpy as np
 
 from smac.scenario.scenario import Scenario
 from smac.tae.execute_ta_run import StatusType
+from smac.tae.execute_ta_run_old import ExecuteTARunOld
 from smac.runhistory.runhistory import RunHistory
 from smac.stats.stats import Stats
 from smac.optimizer.objective import average_cost
 from smac.utils.io.traj_logging import TrajLogger
-from smac.utils.validate import Validator
+from smac.utils.validate import Validator, Run
+
+if sys.version_info[0] == 2:
```

Comment:
why is this important? We don't support Python 2 anymore.

Reply:
True.